### PR TITLE
feat(growth): Sending analytics for admin requests

### PR DESCRIPTION
**Pull request by PixelDust22**
Add an analytics event on the backend for each time we send a request email for Invite Requests

### CODE REVIEW

File: src/sentry/analytics/events/admin_request.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+class InviteRequestSent(analytics.Event):
+    type = "admin_request.sent"
+    attributes = (
+        analytics.Attribute("organization_id"),
+        analytics.Attribute("user_id"),
+        analytics.Attribute("target_user_id"),
+        analytics.Attribute("providers"),
+        analytics.Attribute("ty"),
```
Comment (scefali) [inline src/sentry/analytics/events/admin_request.py:11]: @Neo-Zhixing What does `ty` represent?

Comment (Neo-Zhixing) [reply src/sentry/analytics/events/admin_request.py:11]: ty means type. I had problems naming it `type` so I just used an acronym.

File: src/sentry/analytics/events/admin_request.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from sentry import analytics
+
+
+class InviteRequestSent(analytics.Event):
```
Comment (scefali) [inline src/sentry/analytics/events/admin_request.py:4]: @Neo-Zhixing So we should make a class called `InAppRequest` or something that has the attributes we want then have `InviteRequestSent` and the other events subclass it where each set their own `type` field

Comment (Neo-Zhixing) [reply src/sentry/analytics/events/admin_request.py:4]: I see.

File: src/sentry/tasks/members.py [hunk 3]
```diff
@@ -63,3 +63,10 @@ def send_invite_request_notification_email(member_id):
         except Exception as e:
             logger = get_logger(name="sentry.mail")
             logger.exception(e)
+        analytics.record(
+            "invite_request.sent",
+            organization_id=om.organization.id,
+            user_id=member_id,
```
Comment (scefali) [inline src/sentry/tasks/members.py:69]: @Neo-Zhixing So the `member_id` is not the same as the user id, you should use `om.user_id`

Comment (Neo-Zhixing) [reply src/sentry/tasks/members.py]: turns out it should really be om.inviter.id

**Review by scefali — APPROVED** [review]

**Pull request merged**